src/main/java/au/com/sealink/quicktravel/client/models/Booking.java

```diff
@@ -3,14 +3,62 @@
 import com.google.gson.annotations.Expose;
 import com.google.gson.annotations.SerializedName;
 
+import java.util.Date;
+
 public class Booking {
     @SerializedName("id")
     @Expose
     private int id;
+
     @SerializedName("state")
     @Expose
     private String state;
 
+    @SerializedName("reference")
+    @Expose
+    private String reference;
+
+    @SerializedName("country_id")
+    @Expose
+    private int countryId;
+
+    @SerializedName("public_comments")
+    @Expose
+    private String publicComments;
+
+    @SerializedName("internal_comments")
+    @Expose
+    private String internalComments;
+
+    @SerializedName("post_code")
+    @Expose
+    private String postCode;
+
+    @SerializedName("external_identifier")
+    @Expose
+    private String externalIdentifier;
+
+    @SerializedName("gross_in_cents")
+    @Expose
+    private int grossInCents;
+
+    @SerializedName("nett_in_cents")
+    @Expose
+    private int netInCents;
```

Comment:
I think you either need another t in nett, or vice versa. Whatever is consistent with QT